openmetadata-ui/src/main/resources/ui/src/pages/service/index.tsx

```diff
@@ -475,6 +483,26 @@ const ServicePage: FunctionComponent = () => {
       });
   };
 
+  const fetchContainers = async (paging?: string) => {
+    setIsLoading(true);
+    try {
+      const response = await getContainers(
+        serviceFQN,
+        ['owner', 'tags'],
+        paging
+      );
+
+      setData(response.data);
+      setPaging(response.paging);
+      setInstanceCount(response.paging.total);
```

Comment:
Remove this state variable, We can access it from paging state

Reply:
Sure, I will have a look.